community/modules/compute/htcondor-execute-point/README.md

```diff
@@ -12,9 +12,55 @@ modules.
 [htcondor-install]: ../../scripts/htcondor-install/README.md
 [htcondor-configure]: ../../scheduler/htcondor-configure/README.md
 
+### Known limitations
+
+This module may be used exactly 1 or 2 times in a blueprint to create sets of
+execute points in an HTCondor pool. If using 1 set, it may use either Spot or
+On-demand pricing. If using 2 sets, one must use Spot and the other must
+use On-demand pricing. If you do not follow this constraint, you will likely
+receive an error while running `terraform apply` similar to that shown below.
+Future development is planned to support more heterogeneity within a pool.
```

Comment:
nit. Do you I understand this line correctly?
> Future development is planned to support "Spot & Spot"  or "On-demand  & On-demand" combinations.

Should it be homogeneity instead heterogeneity ?

Reply:
How about "Future development is planned to support more than 2 sets of VM configurations, including all combinations of pricing options."?

Reply:
SGTM